backend-project/small_eod/institutions/serializers.py

```diff
@@ -5,62 +5,65 @@
     Institution,
 )
 from rest_framework import serializers
-
 from ..generic.serializers import UserLogModelSerializer
 
 
 class AdministrativeUnitSerializer(serializers.ModelSerializer):
```

Comment:
Proponuje przenieść to do osobnej aplikacji, bo w #120 będziemy mieli także odpowiadające ViewSets.

Reply:
Zrobione